pkg/controller/gitopsconfig/controller.go

```diff
@@ -282,14 +284,14 @@ func (r *Reconciler) createCronJob(instance *gitopsv1alpha1.GitOpsConfig) error
 			update = false
 		} else {
 			// Error reading the object - requeue the request.
-			return err
+			return xerrors.Errorf("cannot read kubernetes object: %w", err)
 		}
 	}
 
 	err = controllerutil.SetControllerReference(instance, &cronjob, r.scheme)
 	if err != nil {
 		log.Error(err, "unable to the owner for cronjob", "cronjob", cronjob)
-		return err
+		return xerrors.Errorf("unable to the owner for cronjob %q: %w", cronjob, err)
```

Comment:
please improve as mentioned above for similar line

Reply:
Fixed, please let me know if new error message sounds better.